core/genesis.go

```diff
@@ -549,8 +579,17 @@ func (g *Genesis) toBlockWithRoot(root common.Hash) *types.Block {
 			head.RequestsHash = &types.EmptyRequestsHash
 			requests = make(types.Requests, 0)
 		}
+		// If Isthmus is active at genesis, set the WithdrawalRoot to the storage root of the L2ToL1MessagePasser contract.
+		if g.IsIsthmus() {
+			if storageRootMessagePasser == (common.Hash{}) {
+				// if there was no MessagePasser contract storage, set the WithdrawalsHash to the empty hash
```

Comment:
The specs [note](https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/exec-engine.md#backwards-compatibility-considerations) that `After Isthmus activation, if there was never any withdrawal contract storage, a MPT root of an empty list can be set as the withdrawalsRoot`.